### PR TITLE
[plugin.audio.somafm] 2.0.1

### PR DESCRIPTION
**Pull request by evinr**
### Description
v2.0.1 (2021-11-16)

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by RacerXnyc** [comment]
Not sure if this is the right place to put this, and you probably know this already, lol.... but-  It works!

**Comment by evinr** [comment]
@RacerXnyc Any assistance navigating the process would be helpful. This my second day on the job.

**Comment by RacerXnyc** [comment]
Hey @evinr  I don't think I'm going to be much help, I'm not familiar with how things are done.  Took me 20 minutes to figure where to post my 'it works' message.  I can be moral support though, and you're doing great!

:-)

**Review by inverse — APPROVED** [review]

File: plugin.audio.somafm/addon.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Over 30 unique channels of listener-supported, commercial-free, underground/alternative radio broadcasting from San Francisco. All music hand-picked by SomaFM's award-winning DJs and music directors.
+
+Please consider donating at https://somafm.com to keep this service alive!
+        ]]></description>
+        <description lang="de"><![CDATA[
+Über 30 einzigartige Sender vom hörergetragenen, werbefreien, underground / alternative Radio aus San Francisco. Die gesamte Musik ist handerlesen von SomaFMs ausgezeichneten DJs und Musikprogrammleitern.
+
+Bitte denk über eine Spende auf https://somafm.com nach, um diesen Dienst am Leben zu halten!
+        ]]></description>
+        <description lang="nl"><![CDATA[
```
Comment (basrieter) [inline plugin.audio.somafm/addon.xml:28]: `<![CDATA[` is not needed here and in other places.

File: plugin.audio.somafm/addon.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.somafm"
```
Comment (basrieter) [inline plugin.audio.somafm/addon.xml:2]: Please a correct SPDX License identifer (https://spdx.org/licenses/) in a `<license>` tag.

File: plugin.audio.somafm/addon.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.somafm"
+       name="SomaFM"
+       version="2.0.1"
+       provider-name="Soma-FM-Kodi-Add-On Organization">
```
Comment (basrieter) [inline plugin.audio.somafm/addon.xml:5]: The original add-on is from @Oderik, so he/she should approve for this Matrix version.

File: plugin.audio.somafm/lib/channel.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
+import os
```
Comment (basrieter) [inline plugin.audio.somafm/lib/channel.py:1]: Why is this file not in `/resources/lib/`?

File: plugin.audio.somafm/resources/lib/addon.py [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+
+def log(msg):
+    xbmc.log(str(msg), level=xbmc.LOGDEBUG)
+
+
+log(sys.argv)
+
+rootURL = "https://somafm.com/"
+tempdir = xbmc.translatePath("special://home/userdata/addon_data/%s" % __addonid__)
```
Comment (basrieter) [inline plugin.audio.somafm/resources/lib/addon.py:35]: use `xbmcvfs.translatePath` here. `xbmc.translatePath` is no longer supported.

**Review by basrieter — CHANGES_REQUESTED** [review]
Please review my comments.

File: plugin.audio.somafm/addon.xml (hunk 3 above)
Comment (inverse) [reply plugin.audio.somafm/addon.xml:5]: The original addon author (@Oderik) doesn't maintain the previous version of this addon anymore (see: https://github.com/Oderik/xbmc-somafm/issues/13#issuecomment-852879925)

But granted us permission to take over their code base and run it under and organisation GitHub account (see: https://github.com/Oderik/xbmc-somafm/issues/13#issuecomment-970142899)

What other kind of permission would be required?

Thanks for detailed feedback for helping us move this forward 🙏🏼

**Comment by inverse** [comment]
@evinr how comes you closed this PR? Can't you apply the changes that were made on https://github.com/Soma-FM-Kodi-Add-On/plugin.audio.somafm/pull/7 into this PR so we dont lose the PR feedback/context?

**Comment by evinr** [comment]
> @evinr how comes you closed this PR? Can't you apply the changes that were made on [Soma-FM-Kodi-Add-On/plugin.audio.somafm#7](https://github.com/Soma-FM-Kodi-Add-On/plugin.audio.somafm/pull/7) into this PR so we dont lose the PR feedback/context?

@inverse  I will be submitting a new PR with the changes since the best practices for getting addons is to only have a single commit. Sorry for not doing them in succession, got busy today. Once it is submitted I will as for Oderik to make the approval and then link to this PR. Hopefully by this time tomorrow!

**Comment by inverse** [comment]
@evinr don't worry about time commitments, the majority of us are pushing OSS in our spare time. Interesting, I wonder why they have that rule. Can't they squash on merge so that it shows up on master as a single commit?

**Comment by evinr** [comment]
New MR: https://github.com/xbmc/repo-plugins/pull/3798
🤞

**Pull request closed without merging**